search/search.go

```diff
@@ -148,3 +215,8 @@ type SearcherOptions struct {
 type SearchContext struct {
 	DocumentMatchPool *DocumentMatchPool
 }
+
+func (sc *SearchContext) SizeOverhead() int {
```

Comment:
is there a reason this is named SizeOverhead and not Size like the rest?

Reply:
Oh, let me update these, I missed a TODO I had to complete here.